### PR TITLE
Update /partners/desktop for canonical-v3

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**note: the design differs from the [copydoc](https://docs.google.com/document/d/1MBVH4kou0Q7yO_Lq6DPQpZL_rc0K1-nVfvg4aF3biSc/edit#) a little**
- Updates /partners/desktop with the new style of canonial-v3, see [design](https://app.zeplin.io/project/624ed20e0af92866116ea26d/screen/625418364f034e676f297348)
- Adds a `p-card.has-cta` class 

## QA

- Compare against design

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5154

### CODE REVIEW

File: templates/partners/desktop.html [hunk 1]
```diff
@@ -7,143 +7,171 @@
...
+    <div class="col-12 col-medium-6">
+      <h2 style="margin-bottom: 3rem;">Why buy a certified Ubuntu desktop?</h2>
+    </div>
+    <div class="col-4 col-medium-2 p-card--strip-thin">
+      <h3 class="p-heading--4">Preloaded with Ubuntu</h3>
+      <p>Ubuntu certified PCs ship with a clean version of Ubuntu, straight from Canonical</p>
+    </div>
+    <div class="col-4 col-medium-2 p-card--strip-thin">
+      <h3 class="p-heading--4">Works out of the box</h3>
+      <p>Canonical performs 100s of stringent tests to ensure all Ubuntu functionality works out-of-the-box</p>
```
Comment (mtruj013) [inline templates/partners/desktop.html:42]: On the doc it's "out of the box" instead of "out-of-the-box"

File: templates/partners/desktop.html [hunk 2]
```diff
@@ -7,143 +7,171 @@
...
+          alt="HP",
+          width="80",
+          height="80",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>HP sells a wide range of Canonical-certified Ubuntu laptops and desktop machines globally.</p>
+        <p><a href="">See the Ubuntu certified AI-ready workstations&nbsp;&rsaquo;</a></p>
```
Comment (mtruj013) [inline templates/partners/desktop.html:128]: This isn't on the doc anymore

File: templates/partners/desktop.html [hunk 3]
```diff
@@ -7,143 +7,171 @@
...
+      </div>
+      <a href="https://certification.ubuntu.com/desktop/models?vendors=HP" class="p-button--positive p-card__cta"><span>View HP with Ubuntu</span></a>
```
Comment (mtruj013) [inline templates/partners/desktop.html:130]: Links to https://ubuntu.com/certified?category=Desktop&category=Laptop on the doc

File: templates/partners/desktop.html [hunk 4]
```diff
@@ -7,143 +7,171 @@
...
+          height="40",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>Canonical has been shipping Ubuntu on a variety of Lenovo's popular machines which are available from Lenovo's enterprise teams.</p>
+        <p><a href="https://www.lenovo.com/us/en/think-workstations/artificial-intelligence-industry/c/artificial-intelligence-industry">Buy AI workstations with Ubuntu&nbsp;&rsaquo;</a></p>
+      </div>
+      <a href="https://certification.ubuntu.com/desktop/models?vendors=Lenovo" class="p-button--positive p-card__cta"><span>View Lenovo with Ubuntu</span></a>
```
Comment (mtruj013) [inline templates/partners/desktop.html:147]: Links to https://ubuntu.com/certified?category=Desktop&category=Laptop on the doc

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good @petesfrench! Just a few comments:

File: templates/partners/desktop.html (hunk 4 above)
Comment (petesfrench) [reply templates/partners/desktop.html:147]: I wasn't sure about these as they reference specific brands of laptop but want to link to the generic laptop page. I will ask in the doc.

**Review by mtruj013 — APPROVED** [review]
Just a couple of things I'd missed earlier, everything else looks good 👍🏻

**Pull request merged**